stress/shell/src/main/java/alluxio/stress/cli/StressJobServiceBench.java

```diff
@@ -0,0 +1,280 @@
+        stats.mMaxResponseTimeNs = entry.getValue().getMaxTimeNs();
+        mResult.putStatisticsForMethod(entry.getKey(), stats);
+      }
+    }
+
+    public synchronized JobServiceBenchTaskResult getResult() {
+      return mResult;
+    }
+  }
+
+  private final class BenchThread implements Callable<Void> {
+    private final BenchContext mContext;
+    private final Histogram mResponseTimeNs;
+    private final String mPath;
+    private final JobServiceBenchTaskResult mResult = new JobServiceBenchTaskResult();
+
+    private BenchThread(BenchContext context, String path) {
+      mContext = context;
+      mResponseTimeNs = new Histogram(StressConstants.TIME_HISTOGRAM_MAX,
+          StressConstants.TIME_HISTOGRAM_PRECISION);
+      mPath = path;
+    }
+
+    @Override
+    public Void call() {
+      try {
+        runInternal();
+      } catch (Exception e) {
+        mResult.addErrorMessage(e.getMessage());
+      }
+      // Update local thread result
+      mResult.setEndMs(CommonUtils.getCurrentMs());
+      mResult.getStatistics().encodeResponseTimeNsRaw(mResponseTimeNs);
+      mResult.setParameters(mParameters);
+      mResult.setBaseParameters(mBaseParameters);
+      // merge local thread result with full result
+      mContext.mergeThreadResult(mResult);
+      return null;
+    }
+
+    private void runInternal() throws Exception {
+      // When to start recording measurements
+      mResult.setRecordStartMs(mContext.getStartMs());
+      long waitMs = mContext.getStartMs() - CommonUtils.getCurrentMs();
+      if (waitMs < 0) {
+        throw new IllegalStateException(String.format(
+            "Thread missed barrier. Set the start time to a later time. start: %d current: %d",
+            mContext.getStartMs(), CommonUtils.getCurrentMs()));
+      }
+      CommonUtils.sleepMs(waitMs);
+      long startNs = System.nanoTime();
+      applyOperation(mPath);
+      long endNs = System.nanoTime();
+      mResult.incrementNumSuccess(1);
+      // record response times
+      long responseTimeNs = endNs - startNs;
+      mResponseTimeNs.recordValue(responseTimeNs);
+      long[] maxResponseTimeNs = mResult.getStatistics().mMaxResponseTimeNs;
+      if (responseTimeNs > maxResponseTimeNs[MAX_RESPONSE_TIME_BUCKET_INDEX]) {
+        maxResponseTimeNs[MAX_RESPONSE_TIME_BUCKET_INDEX] = responseTimeNs;
+      }
+    }
+
+    private void applyOperation(String dirPath) throws IOException, AlluxioException {
+      switch (mParameters.mOperation) {
+        case DISTRIBUTED_LOAD:
+          // send distributed load task to job service and wait for result
+          int numReplication = 1;
+          FileSystemContext fsContext =
+              FileSystemContext.create(new InstancedConfiguration(ConfigurationUtils.defaults()));
+          DistributedLoadCommand cmd = new DistributedLoadCommand(fsContext);
+          DistributedLoadUtils.distributedLoad(cmd, new AlluxioURI(dirPath), numReplication,
```

Comment:
do we have a way of checking for failure?

Reply:
use DistributedLoadCommand.getFailedCount() for this info and record in result

Reply:
use numSuccess to record number of loaded files so we can know number of failure as well.